index.html

```diff
@@ -655,27 +655,44 @@ <h4>Amanda Jepson</h4>
           </div>
 
         </div>
-          <div class="row"> 
+          <div class="row">
             <!--new change-->
             <!-- awesome code about me-->
             <!--charlesmotaroki@students.uonbi.ac.ke	-->
+            <div class="col-lg-3 col-md-6 d-flex align-items-stretch">
+              <div class="member" data-aos="fade-up" data-aos-delay="400">
+                <div class="member-img">
+                  <img src="assets/img/team/team-5.jpg" class="img-fluid" alt="">

```

Comment:
Image should be 600px by 600px for the width and height.

Reply:
The image is already 600px x 600px